recipes/meta.yaml

```diff
@@ -0,0 +1,56 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```

Comment:
Please remove all the comments from the example. They are there just to guide you but we don't need that in a final recipe.